src/components/clusterDetail/ClusterDetailsButtonGroup.tsx

```diff
@@ -0,0 +1,57 @@
+import { Flex, FlexItem, Button, ButtonVariant } from '@patternfly/react-core';
+import React from 'react';
+import { Cluster } from '../../api';
+import { canDownloadClusterLogs } from '../hosts/utils';
+import { EventsModalButton } from '../ui/eventsModal';
+import KubeconfigDownload from './KubeconfigDownload';
+import { downloadClusterInstallationLogs } from './utils';
+import { AlertsContext } from '../AlertsContextProvider';
+
+type ClusterDetailsButtonGroupProps = {
+  cluster: Cluster;
+};
+
+const getID = (suffix: string) => `cluster-detail-${suffix}`;
+
+const ClusterDetailsButtonGroup: React.FC<ClusterDetailsButtonGroupProps> = ({ cluster }) => {
```

Comment:
Isn't this group missing Abort installation button?

I'd really like to use this group in Standalone's `ClusterDetail` page and completely get rid of ClusterToolbar.

Reply:
right, added